src/index.html

```diff
@@ -5,8 +5,314 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          src="images/logo.png"
+          alt="MoyoLogo"
+          class="header__logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="list">
+          <li class="list__item">
+            <a href="#apple" class="list__link is-active">Apple</a>
+          </li>
+          <li class="list__item">
+            <a href="#samsung" class="list__link">Samsung</a>
+          </li>
+          <li class="list__item">
+            <a href="#smartphones" class="list__link">Смартфоны и телефоны</a>
+          </li>
+          <li class="list__item">
+            <a
+              href="#pc"
+              class="list__link"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
```

Comment:
`list` should be a `nav` element, and `item, link` - also. This way we could simply reuse nav block. We would have only one file with styles(not `nav and list` -2)
And add changes to styles. `nav.scss` should be a separate file.
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active">Apple</a>
          </li>
          <li class="nav__item">
            <a href="#samsung" class="nav__link">Samsung</a>
          </li>
          <li class="nav__item">
            <a href="#smartphones" class="nav__link">Смартфоны и телефоны</a>
          </li>
          <li class="nav__item">
            <a
              href="#pc"
              class="nav__link"
              data-qa="nav-hover"
            >
              Ноутбуки и компьютеры
            </a>
          </li>
```